lib/praxis/extensions/attribute_filtering/filtering_params.rb

```diff
@@ -207,13 +219,25 @@ def initialize(parsed = [])
           @parsed_array = parsed
         end
   
+        def matching_leaf_filter(filter_string)
+          return nil unless allowed_leaves.keys.presence
```

Comment:
`.presence`? Did you mean `present?`? not sure where `Array#presence` is from.

Reply:
ah, well. it does exist and it is handy (activeSupport), but since we're not using the return value here, it is unnecessary. `.present?` is enough. 👍